src/processors/JsonSchemaInputProcessor.ts

```diff
@@ -74,14 +78,146 @@ export class JsonSchemaInputProcessor extends AbstractInputProcessor {
   }
 
   /**
-     * Simplifies a JSON Schema into a common models
-     * 
-     * @param schema to simplify to common model
-     */
-  static convertSchemaToCommonModel(schema: Schema | boolean) : {[key: string]: CommonModel} {
+   * Reflect name from given schema and save it to `x-modelgen-inferred-name` extension.
+   * 
+   * It should be removed when we'll simplify current solution for processing schema to CommonModel.
+   * 
+   * @param schema to process
+   * @param namesStack
+   * @param name
+   * @param isRoot
```

Comment:
need a definition of the arguments. `isRoot` is pretty obvious, but the other two could use a description IMO

Reply:
🆗